index.Rmd

```diff
@@ -148,33 +144,40 @@ There is an equivalent in binary format called WKB, easier for computers to proc
 > **Question 1**: What does WKB mean? (hint: think about WKT)
 ```
 
+# Geopandas: GeoSeries and GeoDataFrames
 
-## Geopandas: GeoSeries and GeoDataFrames
-
-GeoPandas strives to make vector processing in Python easier and has many functions available for exploratory vector data analysis. GeoPandas is based on Pandas. Pandas has two main data structures: the Pandas Series and the DataFrame. Correspondingly, Geopandas has two main data structures: the GeoSeries and the GeoDataFrame. 
+GeoPandas strives to make vector processing in Python easier and has many functions available for exploratory vector data analysis. GeoPandas is based on Pandas. Pandas has two main data structures: the `Series` and the `DataFrame`. Correspondingly, GeoPandas has two main data structures: the `GeoSeries` and the `GeoDataFrame`. 
 
-A GeoSeries is a vector of features, where each feature contains: 1) an index, and 2) a geometry. The latter is a shapely.geometry object, and therefore inherits attributes and methods from shapely geometries, such as area, bounds, distance, etc. Finally, a GeoSeries can contain a coordinate reference system (crs). Geopandas functions, such as buffering, can be applied to GeoSeries:
+A `GeoSeries` is a vector of features, where each feature contains: 1) an index, and 2) a geometry. The latter is a `shapely.geometry` object, and therefore inherits attributes and methods from shapely geometries, such as area, bounds, distance, etc. Finally, a `GeoSeries` can contain a coordinate reference system (crs). GeoPandas functions, such as buffering, can be applied to `GeoSeries`:
 
 ```{Python, eval=FALSE}
 import geopandas as gpd
 from shapely.wkt import loads
 
 # Define a point
 wktstring = 'POINT(173994.1578792833 444133.6032947102)'
+
 # Convert to a GeoSeries
 gs = gpd.GeoSeries([loads(wktstring)])
+
+# Inspect the properties
 print(type(gs), len(gs))
+
 # Specify the projection
 gs.crs = "EPSG:28992" 
-# Apply function, here a buffer of 100 m
+
+# We can now apply a function
+# As an example, we add a buffer of 100 m
 gs_buffer = gs.buffer(100)
+
+# Inspect the results
 print(gs.geometry)
 print(gs_buffer.geometry)
 ```
 
-A GeoDataFrame is a tabular data structure with multiple columns, where one column is a GeoSeries. GeoDataFrames can be loaded from a file, created with data or loaded from a Pandas dataframe. A Pandas DataFrame is, just like the structured numpy array you learned about in the previous lesson, a dataframe equivalent of R in Python.
+A `GeoDataFrame` is a tabular data structure with multiple columns, where one column is a `GeoSeries`. `GeoDataFrames` can be loaded from a file, created with data or loaded from a Pandas `DataFrame`. A Pandas `DataFrame` is, just like the structured NumPy array you learned about in the previous tutorial, a dataframe equivalent of R in Python.
```

Comment:
Probably useful to also state that a `GeoSeries` is then an equivalent to a `geometry` column/vector in R.